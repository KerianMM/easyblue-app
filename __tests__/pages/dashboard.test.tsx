import '@testing-library/jest-dom';
import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from "../../src/pages/dashboard";

test('<Dashboard/> SNAPSHOT', () => {
    const tree = renderer.create(<Dashboard/>).toJSON();

    expect(tree).toMatchSnapshot();
});
