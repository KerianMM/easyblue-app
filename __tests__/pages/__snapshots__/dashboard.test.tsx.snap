// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard/> SNAPSHOT 1`] = `
<div
  className="container"
>
  <h1
    className="jsx-3144720010"
  >
    Dashboard
  </h1>
</div>
`;
